### PR TITLE
[Bug] Python Interface Microservice accessing attributes

### PR DESCRIPTION
**Pull request by clayandgen**

### CODE REVIEW

**Comment by mcosgriff** [comment]
Lets add some unit tests to cover this edge case.

File: openc3/python/openc3/microservices/interface_microservice.py [hunk 1]
```diff
@@ -259,23 +259,23 @@ def process_cmd(self, topic, msg_id, msg_hash, _redis):
             if msg_hash.get(b"interface_details"):
                 return json.dumps(self.interface.details(), cls=JsonEncoder)
 
-        target_name = msg_hash[b"target_name"].decode()
+        target_name = msg_hash.get(b"target_name", b"").decode()
```
Comment (mcosgriff) [inline openc3/python/openc3/microservices/interface_microservice.py]: We want this to pass instead of raising a useful error message?

File: openc3/python/openc3/microservices/interface_microservice.py [hunk 2]
```diff
@@ -259,23 +259,23 @@ def process_cmd(self, topic, msg_id, msg_hash, _redis):
...
         if target_name and not self.interface.cmd_target_enabled.get(target_name, False):
             return None  # Return and don't ack given target_name if disabled
-        cmd_name = msg_hash[b"cmd_name"].decode()
-        manual = ConfigParser.handle_true_false(msg_hash[b"manual"].decode())
+        cmd_name = msg_hash.get(b"cmd_name", b"").decode()
```
Comment (mcosgriff) [inline openc3/python/openc3/microservices/interface_microservice.py]: We want an empty string downstream?

File: openc3/python/openc3/microservices/interface_microservice.py (hunk 1 above)
Comment (clayandgen) [reply openc3/python/openc3/microservices/interface_microservice.py]: That's a good point -- since this is required, throwing a KeyError here is probably fine

File: openc3/python/openc3/microservices/interface_microservice.py (hunk 2 above)
Comment (clayandgen) [reply openc3/python/openc3/microservices/interface_microservice.py]: That's a good point -- since this is required, throwing a KeyError here is probably fine
